build/changelog.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const path = require('path');
+const DOC_PATH = './doc';
+const conventionalChangelog = require('conventional-changelog');
+const dest = path.resolve(process.cwd(), DOC_PATH, 'CHANGELOG.md');
+
+let changelogChunk = '';
+const changelogStream = conventionalChangelog({ preset: 'angular' })
+  .on('data', chunk => changelogChunk += chunk.toString('utf8'))
```

Comment:
there's no error handler here, might be helpful if things go south

Reply:
This script is not used atm, not sure if we want to stick with npm script or this one?

Reply:
if we don't use it lets remove it. I prefer npm scripts whereever possible